.azure-pipelines/ci-build.yml

```diff
@@ -251,11 +258,6 @@ extends:
                   inputs:
                     versionSpec: '>=6.11.0'
                     checkLatest: true
-                - task: DownloadPipelineArtifact@2
-                  displayName: Download nupkg from artifacts
-                  inputs:
-                    artifact: Nugets
-                    source: current
                 - task: PowerShell@2
                   displayName: 'Extract release information to pipeline'
```

Comment:
We might have to upload the script as an artifact since the release jobs can't/won't checkout sources anymore AFAICT.
https://github.com/microsoft/kiota/pull/5851

Reply:
I believe this is already being done as the step on line 227 copies the scripts to the artifact staging directory....

Reply:
Thanks for clarifying @andrueastman